packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -6,6 +6,8 @@ import * as jsf from 'json-schema-faker';
 // @ts-ignore
 import * as sampler from 'openapi-sampler';
 
+jsf.extend('faker', () => require('faker'));
```

Comment:
Shouldn't this be `x-faker`?

Reply:
Also, can we import faker using `import` and then use the value here directly?

Reply:
1. No. JSON-schema-faker removes all `x-` in the keywords before parsing. [Source](https://github.com/json-schema-faker/json-schema-faker/blob/573317001f7e3154e409f2616ed6bcd62aa03123/src/class/Container.js#L129)
2. https://github.com/stoplightio/prism/pull/384/commits/30f667364e779fe95e9ab4241cf26465750f9715

Reply:
//cc @philsturgeon Do you see anything potentially risky with this?

Reply:
@XVincentX yeah seeing as this is all just part of the output process this seems fine to me.